index.html

```diff
@@ -64,7 +61,186 @@ <h4>LET'S CONNECT</h4>
+        </p>
+        <h3>LET'S CONNECT</h4>
+        <div id="icons">
+          <ul id="list-icons">
+            <li class="img">
+              <a href="#">
+                <img src="./image/tweeter.png" alt="" />
+              </a>
+            </li>
+            <li class="img">
+              <a href="#">
+                <img src="./image/Linkedin icon.jpg" alt="" />
+              </a>
+            </li>
+            <li class="img">
+              <a href="#">
+                <img src="./image/m.png" alt="" />
+              </a>
+            </li>
+            <li class="img">
+              <a href="#">
+                <img src="./image/github.png" alt="" />
+              </a>
+            </li>
+            <li class="img">
+              <a href="#">
+                <img src="./image/hand.png" alt="" />
+              </a>
+            </li>
+          </ul>
+        </div>
+        <button class="section3-button">Get my resume</button>
+        <div class="menu-categories">
+          <ul class="categories">
+            <li class="li-categories">Languages</li>
+            <img class="li-img" src="./image/arrow-down.png" alt="" />
+            <ul class="dropdown-menu">
+              <li>
+                <img src="./image/Ellipse1.png" />
+                JavaScript
+              </li>
+              <li>
+                <img src="./image/Ellipse2.png" />
+                HTML
+              </li>
+              <li>
+                <img src="./image/Ellipse3.png" />
+                CSS
+              </li>
+            </ul>
+          </ul>
+          <ul class="categories">
+            <li class="li-categories">Framekors
+            </li>
+            <img class="li-img" src="./image/Union.png"/>  
+          </ul>
+          <ul class="categories">
+            <li class="li-categories">Skills</li>
+            <img class="li-img" src="./image/Union.png"/>
+          </ul>
+        </div>
+      </section>
+      <section class="section4">
+          <h2>Contact me</h2>
+          <p> If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!!</p>
+
+              <form action="https://formspree.io/f/xyyvabjj" method="post">
+                <ul>
+                  <li>
+                    <input class="name" type="text" placeholder="Lorem Ipsum" maxlength="30" required>
+                  </li>
+                  <li>
+                    <input class="email" type="email" placeholder="loremipsum@gmail.com"  required>
+                  </li>
+                  <li>
+                    <textarea type="textarea" class="textarea" cols="30" rows="10" placeholder="Write your message here..." maxlength="500" required></textarea>
+                  </li>
+                  <li>
+                    <button class="button-form">Get in touch</button>
+                  </li>
+                </ul>

```

Comment:
Please consider making the same padding inside the input elements to make the design exactly in accordance with the Figma design.

### YOUR DESIGN:-

![image](https://user-images.githubusercontent.com/94223367/235895522-927fd97c-ea05-454b-ac8e-df972d332b82.png)


### FIGMA DESIGN:-

![image](https://user-images.githubusercontent.com/94223367/235895434-66aefdde-915a-4434-9392-e7203baa31a1.png)